modules/manage/partials/internal-use-property.adoc

```diff
@@ -0,0 +1 @@
+WARNING: This property is exclusive for internal use. Please do not modify it unless specifically instructed by Redpanda Support.
```

Comment:
I'd phrase it along the lines of:
`WARNING: This property is for Redpanda internal use only. Do not modify it unless specifically instructed by Redpanda Support.`